### DISCUSSION TITLE
Welcome to NeMo-Guardrails Discussions!

### DISCUSSION THREAD

**Discussion by drazvan** [NVIDIA-NeMo/Guardrails#5] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Alokbpandey** [comment]
Hi i am alok pandey,
 I am Juvenile in this field , I have joined this community to explore and learn
